op-signer/tls-client.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Usage:
+# export SIGNER_CLIENT_DNS=<DNS of the client>
```

Comment:
why we need to specify the client dns name?

Reply:
The client DNS is the client's identity during TLS handshakes, which is in the allow list of server's `auth` config.